### ISSUE TITLE
CopyQ Not Pasting in Ubuntu 18.04 (Gnome 3.28.2)

### ISSUE THREAD

**Issue by nicorellius** [hluk/CopyQ#1280]
I love CopyQ, but I have some problems with it now. I recently upgraded from Ubuntu 16.04 to Ubuntu 18.04. With this, I started using Gnome 3. I completely uninstalled and removed all CoyQ packages, then started fresh according to your docs. Now it's not working. The clipboard fills the CopyQ list and the items are there, but when I click them to fill fields (whether in the web browser -- Firefox 71.0 -- or on my system), the data doesn't fill in. I can click an item, the use Ctrl-V to paste, but this defeats the whole purpose.

**Comment by hluk** [comment]
Did you enable the "pasting" options in configuration? (Thought I think these options are enabled by default.)

These command (depending on whether you want to paste from main window or/and tray) should print `true`:

```
copyq config activate_pastes
copyq config tray_item_paste
```

Anything in logs (F12 shortcut)?

**Comment by nicorellius** [comment]
@hluk Thanks for your response.

Both of these command return `true`:

    copyq config activate_pastes
    copyq config tray_item_paste

See attached for log.
[copyq.log](https://github.com/hluk/CopyQ/files/3970745/copyq.log)

Is CopyQ supposed to function in the way I'm expecting? For example, after you fill the list, is it expected that I can simply bring focus to a field and then click the indicator icon, choose and item, click it, and it will appear in the field? Or is it supposed to function by selecting the item in the list and then using Ctrl-V to paste it in the field?

**Comment by hluk** [comment]
> Is CopyQ supposed to function in the way I'm expecting? For example, after you fill the list, is it expected that I can simply bring focus to a field and then click the indicator icon, choose and item, click it, and it will appear in the field?

Yes, this is the default behavior. It should paste (with Shift+Insert) to the focused window (after main window or tray menu closes).

> Or is it supposed to function by selecting the item in the list and then using Ctrl-V to paste it in the field?

This is actually how I use it. I never got used to automatic paste.

---

I've recently written [Why pasting from CopyQ doesn’t work?](https://copyq.readthedocs.io/en/latest/faq.html#why-pasting-from-copyq-doesn-t-work) FAQ section. Maybe you can find it helpful.

You can also try disabling the commands in Command dialog (F6 key) that you don't use. Just uncheck the check boxes in the list next to the unneeded commands.

**Comment by Chealer** [comment]
@nicorellius: Does this affect a current Ubuntu version? If not, do you mind if this ticket is discarded?
If it does, does [the FAQ _Why does pasting from CopyQ not work?_](https://copyq.readthedocs.io/en/latest/faq.html#why-does-pasting-from-copyq-not-work) help?
